### ISSUE TITLE
Character encoding - possible to set it?

### ISSUE THREAD

**Issue by mesic** [ruipgil/scraperjs#65]
I'm trying to parse a website however the characters are read wrong, probably due to the character setting. Is there any way this can be set?

**Comment by ruipgil** [comment]
Yes, ``` scraperjs ``` uses [request](https://github.com/request/request), you should refer back to it.

**Comment by PauloAmaroRamos** [comment]
Instead of a string for the url:
`scraperjs.StaticScraper.create('http://www.somesite.com').scrape(...)`

Use an object with the parameters url and encoding, like this:
`scraperjs.StaticScraper.create({ url: 'http://www.somesite.com', encoding: "binary" }).scrape(...)`
